src/SwedbankPay.Sdk.Infrastructure/PaymentOrders/CurrentPaymentResponseObject.cs

```diff
@@ -0,0 +1,60 @@
+﻿using SwedbankPay.Sdk.PaymentInstruments;
+using System;
+
+namespace SwedbankPay.Sdk.PaymentOrders
+{
+    public class CurrentPaymentResponseObject : IdLink, ICurrentPaymentResponseObject
```

Comment:
We need to get rid of the word `Object` in these ... objects. 😄 

```suggestion
    public class CurrentPayment : IdLink, ICurrentPayment
```